### ISSUE TITLE
Could not import content pack

### ISSUE THREAD

**Issue by aremai** [noktork/Graylog-CISCO-ASA-Extractor#2]
Hi,

I'm receiving this error when trying to import your content pack. Is there a specific requirement for it work, or did I miss something?

```
Could not import content pack
Error importing content pack, please ensure it is a valid JSON file. Check your Graylog logs for more information.
```

cheers,
theresa

**Comment by webmastir** [comment]
I encountered the same problem. I then realized it should be done under the "Import extractors to Input" section.

1. Under /system/inputs, select Manage extractors for whatever input you created
2. Under Actions, select Import extractors
3. Paste the contents of [asa_extractor.json](https://github.com/noktork/Graylog-CISCO-ASA-Extractor/blob/master/asa_extractor.json) into the field provided
4. Click Add extractors to input

It works great for me. I was able to create some useful metrics with these extractors.
